supportedBackends/google/batch/src/main/scala/cromwell/backend/google/batch/actors/GcpBatchAsyncBackendJobExecutionActor.scala

```diff
@@ -271,15 +271,24 @@ class GcpBatchAsyncBackendJobExecutionActor(override val standardParams: Standar
       }
     )
 
-  override lazy val inputsToNotLocalize: Set[WomFile] = {
-    val localizeOptional = jobDescriptor.findInputFilesByParameterMeta {
-      case MetaValueElementObject(values) => values.get("localization_optional").contains(MetaValueElementBoolean(true))
-      case _ => false
+  // TODO: There is an AWS version of this that looks functionally identical. Consider unifying.
```

Comment:
Worth a ticket in the CTM board? This might be a good onboarding ticket for a future dev